api/v1alpha1/networkchaos_types.go

```diff
@@ -166,6 +166,10 @@ type NetworkChaosSpec struct {
 	// Target represents network target, this applies on netem and network partition action
 	// +optional
 	Target *Target `json:"target,omitempty"`
+
+	// ExternalTargets represents network targets outside k8s
+	// +optional
+	ExternalTargets []string `json:"externalTargets,omitempty"`
```

Comment:
Can we use some way to represent the range of IPs? such as "192.168.0.0/24"

Reply:
It supports CIDR input now :smile: